conda/core/subdir_data.py

```diff
@@ -5,37 +5,33 @@
 import json
 import pickle
 import re
-import warnings
 from collections import UserList, defaultdict
 from contextlib import closing
-from errno import EACCES, ENODEV, EPERM, EROFS
+from errno import ENODEV
 from functools import partial
 from itertools import chain, islice
 from logging import getLogger
 from mmap import ACCESS_READ, mmap
-from os.path import dirname, exists, isdir, join, splitext
+from os.path import exists, join, splitext
 from pathlib import Path
 from time import time
 
 from genericpath import getmtime, isfile
 
-try:
-    from boltons.setutils import IndexedSet
-except ImportError:  # pragma: no cover
-    from .._vendor.boltons.setutils import IndexedSet
-
 from conda.gateways.repodata import (
+    CACHE_STATE_SUFFIX,
     CondaRepoInterface,
     RepodataCache,
+    RepodataFetch,
     RepodataIsEmpty,
-    RepodataOnDisk,
     RepodataState,
     RepoInterface,
-    Response304ContentUnchanged,
     cache_fn_url,
+    create_cache_dir,
+    get_repo_interface,
 )
 
-from .. import CondaError
+from .._vendor.boltons.setutils import IndexedSet
```

Comment:
I thought we were using the external boltons now.

Reply:
This PR started before that change.